automation_tools/__init__.py

```diff
@@ -911,6 +912,8 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     host = env.get('vm_ip', env['host'])
 
     # Register and subscribe machine to Red Hat
+    if test_in_stage:
```

Comment:
Add at the top of the task something like:

```
     if isinstance(test_in_stage, str):
         create_vm = (test_in_stage.lower() == 'true')
```

This is required because fabric pass the argument as an string.